prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -17,6 +17,9 @@ elements:
   - name: patient_suffix
     csvFields: [{name: Patient_suffix}]
 
+  - name: patient_tribal_citizenship
+    csvFields: [{name: Tribal_affiliation}]
```

Comment:
**suggestion:** Can we call this column `Patient_tribal_affiliation`, so it retains the same specificity as the other patient columns?